src/oscar/apps/search/search_handlers.py

```diff
@@ -9,7 +9,43 @@
 FacetMunger = get_class('search.facets', 'FacetMunger')
 
 
-class SearchHandler(object):
+class SearchResultsPaginationMixin(object):
```

Comment:
Please remove the `object` arg, not needed in Python 3.

Reply:
Done.